src/main/java/org/akhq/repositories/RecordRepository.java

```diff
@@ -76,6 +77,8 @@
     @Value("${akhq.clients-defaults.consumer.properties.max.poll.records:50}")
     protected int maxPollRecords;
 
+    private final int maxKafkaMessageLength = 1000000; // 1MB
```

Comment:
Please add `@Value` to be able to configure the limit and add a default on `application.yml`